integ/cleanup.sh

```diff
@@ -66,10 +67,54 @@ delete_stack() {
         log ERROR "Failed to wait for ${stack_name} to delete"
         aws cloudformation describe-stack-events \
             --stack-name "${stack_name}"
+        return
     fi
     log INFO "Cloudformation stack '${stack_name}' deleted!"
 }
 
+delete_services() {
+    local cluster="${1:?}"
+    log INFO "Deleting services running on cluster '${cluster}'"
+    if ! services=$(aws ecs list-services \
+        --cluster ecs-updater-integ-cluster \
+        --query 'serviceArns[]' \
+        --output text); then
+        log ERROR "Failed to list services in cluster '${cluster}'"
+        return
+    fi
+
+    for service in ${services}; do
+        log INFO "Deleting service '${service}'"
+        if ! aws ecs delete-service \
+            --cluster "${cluster}" \
+            --service "${service}" \
+            --force >/dev/null; then
+            log ERROR "Failed to delete service '${service}'"
+        fi
+    done
+}
+
+stop_tasks() {
+    local cluster="${1:?}"
+    log INFO "Stopping tasks running on cluster '${cluster}'"
+    if ! tasks=$(aws ecs list-tasks \
+        --cluster ecs-updater-integ-cluster \
+        --query 'taskArns[]' \
+        --output text); then
+        log ERROR "Failed to list tasks in cluster '${cluster}'"
+        return
```

Comment:
This is an error but is returning successfully.  Do we want to stop execution if we see an error here?

Reply:
My thought process was to just log errors and continue to cleanup as many resources as possible. I am returning on error at places where there is no point in continuing to next step because its dependent on output of current step.